font/grub2font.ksy

```diff
@@ -0,0 +1,100 @@
+meta:
+  id: grub2font
+  title: GRUB2 font
+  file-extension: pf2
+  tags:
+    - font
+  license: CC0-1.0
+  endian: be
+doc: |
+  Bitmap font format for the GRUB 2 bootloader.
+doc-ref: http://grub.gibibit.com/New_font_format
+seq:
+  - id: font_header
+    contents: ['FILE', 0, 0, 0, 4, 'PFF2']
+    size: 12
+  - id: font_sections
+    type: font_section
+    repeat: until
+    repeat-until: _.section_name == "DATA"
+types:
+  font_section:
+     seq:
+       - id: section_name
+         size: 4
+         type: str
+         encoding: ASCII
+       - id: section_length
+         type: u4
+       - id: body
+         size: section_length
+         type:
+           switch-on: section_name
+           cases:
+             '"NAME"': font_name
+             '"FAMI"': font_family_name
+             '"WEIG"': font_weight
+             '"SLAN"': font_slant
+             '"PTSZ"': font_point_size
+             '"MAXW"': maximum_character_width
+             '"MAXH"': maximum_character_height
+             '"ASCE"': ascent_in_pixels
+             '"DESC"': descent_in_pixels
+             '"CHIX"': character_index
+         if: section_name != "DATA"
+  font_name:
+    seq:
+      - id: name
+        type: strz
```

Comment:
Allows garbage after the end of a string. Should we parse it too?

Reply:
@KOLANICH You know my attitude to this: https://github.com/kaitai-io/kaitai_struct_formats/pull/381#discussion_r554371635